examples/basic/version.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.49.0"
+      version = "1.65.0"
```

Comment:
The version should also be updated in the other example and the root version.tf

Reply:
Updated